### PR TITLE
#13373: PyTorch Tracing Sweeps - Eltwise set 2

### LINKED ISSUE #13409: PyTorch Tracing Sweeps - Set 2 ops
PR : https://github.com/tenstorrent/tt-metal/pull/13421

- [x] aten.clamp.default.md
- [x] aten.clamp_min.default.md
- [x] aten.hardswish.default.md
- [x] aten.hardtanh.default.md
- [x] aten.logical_not.default.md
- [x] aten.lt.Scalar.md
- [x] aten.lt.Tensor.md
- [x] aten.mul.Scalar.md
- [x] aten.ne.Scalar.md
- [x] aten.neg.default.md
- [x] aten.sub.Tensor.md
- [x] aten.div.Tensor.md

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Issue : #13373
https://github.com/tenstorrent/tt-metal/actions/runs/11234785739 : passed
**Op Status :** 
Screenshot 2024-10-04 at 10 23 13 PM

Hardtanh : https://github.com/tenstorrent/tt-metal/actions/runs/11364348418
All sweeps : https://github.com/tenstorrent/tt-metal/actions/runs/11363605751
https://github.com/tenstorrent/tt-metal/actions/runs/11363593680

### CODE REVIEW

**Review by mouliraj-mcw — APPROVED** [review]

**Review by Aswinmcw — APPROVED** [review]
LGTM

File: tests/sweep_framework/sweeps/eltwise/unary/neg/neg_pytorch2.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    device,
+) -> list:
+    data_seed = random.randint(0, 20000000)
+    torch.manual_seed(data_seed)
+
+    torch_input_tensor_a = gen_func_with_cast_tt(
+        partial(torch_random, low=-100, high=100, dtype=torch.float32), input_dtype
+    )(input_shape)
+
+    torch_output_tensor = torch.neg(torch_input_tensor_a)
```
Comment (umadevimcw) [inline tests/sweep_framework/sweeps/eltwise/unary/neg/neg_pytorch2.py:58]: General comment is that see if we can make use of golden function

**Review by umadevimcw — APPROVED** [review]

**Review by sjameelTT — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]
Pls run sweeps pipeline before merging

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11364348418 - passed for hardtanh

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11364319866 - passed

**Pull request merged**